microsoft-edge/puppeteer.md

```diff
@@ -43,35 +45,88 @@ const puppeteer = require('puppeteer-core');
+Change `executablePath` to point to your installation of Microsoft Edge \(Chromium\).  For example, on macOS, the `executablePath` for Microsoft Edge Canary should be set to `/Applications/Microsoft\ Edge\ Canary.app/`.  To find the `executablePath`, navigate to `edge://version`.  Save your changes.  
 
 > [!NOTE]
-> Microsoft Edge (EdgeHTML) does not work with `puppeteer-core`. Install the Microsoft Edge preview channels, which leverage Chromium, [here](https://www.microsoftedgeinsider.com/download) to continue following this example if you haven't already.
+> Microsoft Edge \(EdgeHTML\) does not work with `puppeteer-core`.  [Install the Microsoft Edge preview channels][MicrosoftedgeinsiderDownload], which leverage Chromium, to continue following this example if you have not already.  
 
-Now run **example.js** from the command line:
+Now run `example.js` from the command-line.  
 
-```console
+```shell
 node example.js
-```
+```  
+
+`puppeteer-core` launches Microsoft Edge, navigates to `https://www.microsoftedgeinsider.com` and saves an 800x600px screenshot of the page.  You are able to customize the page size with [page.setViewport()][PuppeteerApipagesetviewport].  
 
-`puppeteer-core` launches Microsoft Edge, navigates to `https://www.microsoftedgeinsider.com/` and saves an 800x600px screenshot of the page. You are able to customize the page size with [`Page.setViewport()`](https://pptr.dev/#?product=Puppeteer&version=v2.0.0&show=api-pagesetviewportviewport).
+:::image type="complex" source="./media/puppeteer-example.png" alt-text="The example.png file produced by example.js":::
+   Figure 1:  The `example.png` file produced by `example.js`  
+:::image-end:::  
 
+<!--  
 > ##### Figure 1  
-> The **example.png** file produced by **example.js**
+> The `example.png` file produced by `example.js`  
 > ![The example.png file produced by example.js](./media/puppeteer-example.png)  
+-->  
+
+This is just a simple example of the automation and testing scenarios enabled by Puppeteer and `puppeteer-core`.  For more information about Puppeteer and how it works, see [Puppeteer][PuppeteerMain].  
+
+## Feedback  
+
+The Edge Developer team is eager to hear your feedback about using Puppeteer, `puppeteer-core`, and Microsoft Edge!  Use the **Feedback** icon in the Microsoft Edge DevTools or tweet [@EdgeDevTools][TwitterIntentTweetEdgedevtools] to let the Microsoft Edge team know what you think.  
+
+
+:::image type="complex" source="./devtools-guide-chromium/media/devtools-feedback.png" alt-text="The Feedback icon in the Microsoft Edge DevTools":::
+   Figure 2:  The **Feedback** icon in the Microsoft Edge DevTools  
+:::image-end:::  
+
+<!--  
+> ##### Figure 2  
+> The **Feedback** icon in the Microsoft Edge DevTools  
+> ![The Feedback icon in the Microsoft Edge DevTools](./devtools-guide-chromium/media/devtools-feedback.png)  
+-->  
+
+## See also  
```

Comment:
Since some of these links are mentioned in the article, can we reduce the length of the See Also list?

Reply:
Removed redundant links.